loader:
  pcds_dir: "/mnt/d/dataset/semantic_kitti/00/velodyne"
  pose_file: "/mnt/d/dataset/semantic_kitti/00/poses.txt"
  kitti_calibration_file: "/mnt/d/dataset/semantic_kitti/00/calib.txt"
  start: 4390
  end: 4531
  format: "kitti" #"erasor"

map_cleaner:
  save_dir: "/mnt/d/dataset/semantic_kitti/00/mapcleaner"
  frame_id: "map"
  in_process_visualization: true
  visualization_vg_res: 0.1

ground_segmentation:
  use_object_seg: false
  use_voxel_grid: false
  voxel_leaf_size: 0.05

grid_map_builder:
  grid_res: 0.2

variance_filter:
  variance_th: 0.01
  min_num: 1

first_bgk:
  kernel_radius: 1.0
  lambda: 0.0
  sigma: 1.0
  min_num: 3

trajectory_filter:
  normal_th_deg: 15.0

second_bgk:
  kernel_radius: 2.0
  lambda: 0.0
  sigma: 0.5
  min_num: 3

median:
  enable: true
  kernel_size: 3

divide_by_terrain:
  threshold: 0.172
  on_terrain_only: true

moving_point_identification:
  voxel_leaf_size: 0.05
  fov_h_deg: 360.0
  fov_v_deg: 30.0
  res_h_deg: 0.16
  res_v_deg: 0.4
  lidar_range: 70.0
  delta_h: 2
  delta_v: 1
  threshold: 0.5
  frame_skip: 0
  submap_update_dist: 10.0
